platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/roster/RosterUtils.java

```diff
@@ -49,7 +70,92 @@ public static String formatNodeName(final long nodeId) {
      * @return its Hash
      */
     @NonNull
-    public static Hash hash(@NonNull final Roster roster) {
+    public static Hash hashOf(@NonNull final Roster roster) {
         return PBJ_RECORD_HASHER.hash(roster, Roster.PROTOBUF);
     }
+
+    /**
+     * Generates the initial active roster based on the given software version and initial state.
+     * The active roster is obtained by adopting the candidate roster if a software upgrade is detected.
+     * Otherwise, the active roster is created from the address book.
+     *
+     * @param version the software version of the current node
+     * @param initialState the initial state of the platform
+     * @param addressBook  the address book being used by the network
```

Comment:
It's still unclear to me if this is the address book from the config.txt or the address book from the initial state.

Reply:
I initially deliberately left it named as "bootstrapAddressBook" which indicates it's from config.txt but you asked to just name it 'addressBook'. 
It's the address book loaded from config. Eventually in production, this method will not be called by Main classes - there's a pending work where we have to provide the roster based on network mode - bootstrap, genesis etc.

Reply:
My [comment](https://github.com/hashgraph/hedera-services/pull/15442/files/2a9af7a6500b2ddd0aac6dc81835d85668d0ca4c#r1763432819) asking that the generic term `address book` be used was on the generic method `createRoster()`. It does not make sense to be specific about which address book a generic method operates on. This method, `constructInitialRoster()` is quite specific. It is called at a specific point in the lifecycle and does a very specific thing, which warrants specific descriptions of the inputs.